aiida/orm/data/cif.py

```diff
@@ -93,7 +103,7 @@ def symop_string_from_symop_matrix_tr(matrix, tr=(0, 0, 0), eps=0):
 
 
 @optional_inline
-def _get_aiida_structure_ase_inline(cif, parameters):
+def _get_aiida_structure_ase_inline(cif, parameters=None):
```

Comment:
defaults with non database objects such as None (here, parameters=None) should be avoided...